### DISCUSSION TITLE
a simple model in vensim

### DISCUSSION THREAD

**Discussion by sanket-mishra-profile** [quaquel/EMAworkbench#330] [Q&A] [answered]
I tried to run simple model in vensim example given on your webpage, 
First, I have created a vensim file named simple
Then I tried to run it on jupyter notebook
but I could not run the file .. 
It showed me an error - 'vensim dll not found'
could you please help me solve the issue?
![error file](https://github.com/quaquel/EMAworkbench/assets/86866430/755cab03-991c-4896-9375-60c32b74362d)

**Comment by quaquel** [comment, marked as answer]
You need Vensim DSS

**Reply by sanket-mishra-profile** [reply]
Thanks. Yes, I was using PLE.

**Comment by EwoutH** [comment]
Since this is likely not an issue with the workbench, I'm going to convert this to a Discussion.
